### PR TITLE
[NEW] Add Voxtelesys to list of SMS providers

### PR DESCRIPTION
**Pull request by john08burke**
Hello!

I made some mods to integrate Voxtelesys into the list of livechat SMS providers. The API uses Bearer authentication and I implemented the /api/v1/sms method for outbound messaging.

Not sure who to @ to review?

vts_sms

### CODE REVIEW

File: app/sms/server/services/voxtelesys.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+		returnData.media = []; /* MMS not currently supported */
+
+		return returnData;
+	}
+	send(fromNumber, toNumber, message) {
+		const options = {
+			timeout: 30000,
+			followRedirect: false,
+			url: 'https://servicelayer01.voxtelesys.net:3100/api/v1/sms',
```
Comment (graywolf336) [inline app/sms/server/services/voxtelesys.js]: Can you maybe make this url configurable in the event they update it in the future?

Comment (john08burke) [reply app/sms/server/services/voxtelesys.js]: Yes. I can add to the SMS setting.js if this is acceptable? Default will be existing URL.
Something like below:
sms_configs

Comment (john08burke) [reply app/sms/server/services/voxtelesys.js]: @graywolf336 changes are below.

**Review by sampaiodiego — CHANGES_REQUESTED** [review]
from [their docs](https://smsapi.voxtelesys.com/#messages) looks like they use an different authentication: then you're proposing: `authorization: Token token=176a78b0-8e81-4e56-9e3f-da67d17915bb` 

also how do you create this token? can you please provide an explanation for other on how to set this integration up? thanks

**Comment by john08burke** [comment]
The docs have been updated. The `Token token=<uuid>` was the original method of authentication, which mirrored an OAuth2.0-style of authentication. Traditional Bearer authentication was added awhile ago for simplicity. The proposed code opted for the simplistic Bearer approach.

Tokens are assigned upon account creation, [here](https://portal.voxtelesys.net). Right now this is a manual assignment from Voxtelesys after the user has created an account, but will be automated soon.

**Comment by sampaiodiego** [comment]
alright, thanks @john08burke .. lgtm =)

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**